packages/back-end/src/integrations/Mssql.ts

```diff
@@ -78,53 +66,14 @@ export default class Mssql extends SqlIntegration {
   formatDateTimeString(col: string): string {
     return `CONVERT(VARCHAR(25), ${col}, 121)`;
   }
-  async getInformationSchema(): Promise<InformationSchema[]> {
-    const databaseName = this.params.database;
-
-    if (!databaseName)
-      throw new MissingDatasourceParamsError(
-        "To view the information schema for a MS Sql dataset, you must define a default database. Please add a default database by editing the datasource's connection settings."
+  getInformationSchemaFromClause(): string {
+    if (!this.params.database)
+      throw new Error(
```

Comment:
This was a `MissingDatasourceParamsError` before.  Is that still relevant?

Reply:
Updated to go back to `MissingDatasourceParamsError`.